sms_sponsorship/static/src/css/custom.css

```diff
@@ -0,0 +1,58 @@
+@keyframes show_details_button {
```

Comment:
Remove all classes that are not used in your template. You basically just need donation_success, thank_you_container and quote.